### PR TITLE
Fix user-agent string RFC 2616 Section 14.43

### LINKED ISSUE #169: User-agent string warning with ':'
Default user-agent string in got contains ':' character. But many servers shows warnings that this character not allowed in user-agent string

### PR DESCRIPTION
**Pull request by birhoff**
Fixes #169.

### CODE REVIEW

**Comment by floatdrop** [comment]
@birhoff can you fix the tip [here](https://github.com/sindresorhus/got#tip) too?

File: index.js [hunk 1]
```diff
@@ -202,7 +203,7 @@ function normalizeArguments(url, opts) {
 	);
 
 	opts.headers = Object.assign({
-		'user-agent': 'https://github.com/sindresorhus/got',
+		'user-agent': `sindresorhus-got/${version} (https://github.com/sindresorhus/got)`,
```
Comment (sindresorhus) [inline index.js]: Actually, since we include the URL, we can remove `sindresorhus-` from the first part.

**Comment by birhoff** [comment]
this is not that one? https://github.com/sindresorhus/got/pull/171/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R269

**Comment by floatdrop** [comment]
@birhoff oops :blush:

**Comment by birhoff** [comment]
:up:

**Pull request closed without merging**